shared/src/utils/stats/SurveyStatsCalculator.ts

```
@@ -0,0 +1,131 @@
+		this._data = data;
+		this._aggregatedData = aggregatedData;
+		this._oldestDate = oldestDate;
+	}
+
+	/**
+	 * Builds a new instance of SurveyStatsCalculator
+	 * @param firestoreAdmin Firestore admin instance for accessing the database
+	 */
+	static async build(firestoreAdmin: FirestoreAdmin): Promise<SurveyStatsCalculator> {
+		const recipients = await firestoreAdmin.collection<Recipient>(RECIPIENT_FIRESTORE_PATH).get();
+		const surveysData = await this.fetchAndProcessSurveys(firestoreAdmin, recipients);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)